frontend/src/pages/CycleCountBatchDetailPage.jsx

```
@@ -0,0 +1,519 @@
+    // Apply status filter
+    if (statusFilter !== 'all' && item.status !== statusFilter) {
+      return false;
+    }
+    
+    // Apply search filter
+    const searchLower = searchTerm.toLowerCase();
+    return (
+      item.item_type.toLowerCase().includes(searchLower) ||
+      item.item_name.toLowerCase().includes(searchLower) ||
+      item.location.toLowerCase().includes(searchLower) ||
+      (item.assigned_to && item.assigned_to.toLowerCase().includes(searchLower))
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)